src/client/dfuse/ops/lookup.c

```diff
@@ -23,19 +23,12 @@ dfuse_reply_entry(struct dfuse_projection_info *fs_handle,
 	D_ASSERT(ie->ie_parent);
 	D_ASSERT(ie->ie_dfs);
 
-	/* Set the caching attributes of this entry, but do not allow
-	 * any caching on fifos.
-	 */
-
-	if (S_ISFIFO(ie->ie_stat.st_mode)) {
-		if (!is_new) {
-			ie->ie_stat.st_mode &= ~S_IFIFO;
-			ie->ie_stat.st_mode |= S_IFDIR;
-		}
-	} else {
-		entry.attr_timeout = ie->ie_dfs->dfs_attr_timeout;
+	/* Do not cache directly attributes as this does not work with uns */
```

Comment:
Minor typo:
```suggestion
	/* Do not cache directory attributes as this does not work with uns */
```

Reply:
I'm keen to move forward so I'll address this in a future PR.